tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py

```diff
@@ -28,9 +28,9 @@ def search(self, term):
         import time
         time.sleep(10)
 
-        search_toggle = Wait(self.marionette).until(
+        search_toggle = Wait(self.marionette, timeout=30, interval=5).until(
             expected.element_present(*self._search_toggle_locator))
-        Wait(self.marionette).until(expected.element_displayed(search_toggle))
+        Wait(self.marionette, timeout=30, interval=5).until(expected.element_displayed(search_toggle))
```

Comment:
Nothing really. Probably, this change isn't necessary, but when I tried to fix this issue, I had networking issues. I'll revert this change.